src/main/java/picard/util/BedToIntervalList.java

```diff
@@ -116,6 +117,11 @@ protected int doWork() {
         IOUtil.assertFileIsReadable(INPUT);
         IOUtil.assertFileIsReadable(SEQUENCE_DICTIONARY);
         IOUtil.assertFileIsWritable(OUTPUT);
+
+        if(PicardHtsPath.isOther(new PicardHtsPath(INPUT))) {
```

Comment:
Can you switch to checking explicitly for /dev/stdin, instead of relying on this problematic `isOther()` method?

Reply:
Done